pkgs/tools/misc/cp210x-program/default.nix

```
@@ -0,0 +1,33 @@
+{ lib
+, python3
+, fetchFromGitHub
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "cp210x-program";
+  version = "0.4.1";
+
+  src = fetchFromGitHub {
+    owner = "VCTLabs";
+    repo = "cp210x-program";
+    rev = "refs/tags/${version}";
+    sha256 = "sha256-IjKshP12WfFly9cPm6svD4qZW6cT8C7lOVrGenSqbfY=";
+  };
+
+  propagatedBuildInputs = with python3.pkgs; [
+    hexdump
+    pyusb
+  ];
+
+  postInstall = ''
+    cp -sv $out/bin/cp210x-program{.py,}
```

```suggestion
    ln -s $out/bin/cp210x-program{.py,}
```
cp -s is rather weird, lets use ln

---

done, the thing is `cp -s` actually has a memorable argument order while `ln -s` is impossible to remember. they're both in coreutils also. there's no difference except this could've been merged already.